source/includes/harvest/_eeoc.md

```diff
@@ -0,0 +1,80 @@
+# EEOC
+
+## The EEOC object
+
+Every application may have zero or one EEOC object.
+
+```json
+{
+  "application_id": 286,
+  "candidate_id": 406,
+  "race": {
+    "id": 7,
+    "description": "Two or More Races"
+  },
+  "gender": {
+    "id": 1,
+    "description": "Male"
+  },
+  "veteran_status": {
+    "id": 3,
+    "message": "I don't wish to answer"
+  },
+  "disability_status": {
+    "id": 1,
+    "description": "Yes, I have a disability (or previously had a disability)"
+  },
+  "submitted_at": "2017-01-29T15:09:46.806Z"
+}
+```
+
+### Noteworthy attributes
```

Comment:
I didn't mention anything about the actual race, gender, veteran_status, etc fields because I wasn't sure what to say. Any suggestions? Or maybe we don't need to mention anything.